css/styles.css

```diff
@@ -268,3 +268,71 @@ textarea {
   resize: vertical;
   height: 170px;
 }
+
+@media (min-width: 500px) {
+  body {
+    background-color: #172b4d;
+  }
```

Comment:
- [ ] Great job implementing the `media` query, however the width should be between **992px - 1100px** as specified in the requirements.
![image](https://user-images.githubusercontent.com/59416136/154523240-c1b96c3a-9bbe-46e0-9e3a-823124a84846.png)

Kindly fix that.